content/v0.80.0.md

```diff
@@ -0,0 +1,478 @@
+
+- Starting with version 0.80, the configuration file is written in the lauguage
+  in which DOSBox Staging was started.
+- To write an English configuration file, launch with the `-lang en`
+  command-line flag (to start DOSBox Staging in English) and then (re-)write
+  your configuration file.
+
+
+##  4. <a name='Newfeatures'></a>New features
+
+
+###  4.1. <a name='ReelMagicsupport'></a>ReelMagic support
+
+DOS ReelMagic games are now playable thanks to Jon Dennis's
+(patch)[https://github.com/jrdennisoss/dosboxrm] with help from Chris Guthrie
+and Joseph Whittaker.
+
+See the `[reelmagic]` section in your configuration file to enable this. We
+suggest only enabling ReelMagic emulation on an as-needed basis.
+
+<video controls width=640>
+  <source src="/reelmagic_rtz.mp4" type="video/mp4">
+  Your browser does not support the <code>video</code> element.
+</video>
+
+Jon plans to add DMA support (primarily used by the ReelMagic Win 3.1 driver and
+games) when time permits, but most ReelMagic games for DOS are already fully
+functional and playable.
+
+###  4.2. <a name='Mouseimprovements'></a>Mouse improvements
+
+**Dedicated configuration section:**
+
+The mouse configuration has been moved to its own section and additional options
+have been added. A tool called `mousectl` is available on `Z:\>` for more
+detailed per-interface mouse configuration."
+ 
+**Dual mouse gaming:**
+
+Multiple physical mice can now be mapped to specific emulated mouse interfaces,
+enabling two-player split-screen gaming (The Settlers, The Settlers II) using
+two USB mice on a single computer.
+
+<video controls width=640>
+  <source src="/dual_mouse_s2.mp4" type="video/mp4">
+  Your browser does not support the <code>video</code> element.
+</video>
+
+See the (wiki article)[https://github.com/dosbox-staging/dosbox-staging/wiki/Dual-Mouse-Gaming] for more details.
+
+
+**Improved support for 3rd party INT33 Windows mouse driver:**
+ 
+@javispedro's (INT33 mouse
+driver)[https://git.javispedro.com/cgit/vbados.git/about/] for Windows 3.1x is
+fully supported and no longer requires the use of the separate DOS driver,
+except when booting from a disk image. The driver also resolves compatibility
+issues with seamless mouse integration and adds mouse wheel support.
+
+See the (wiki article)[https://github.com/dosbox-staging/dosbox-staging/wiki/Windows-3.1x#improved-mouse-driver] for more details.
+
+**Mouse-related help in the title bar:**
+
+The title bar now displays some short help about the mouse, such as how it can
+be captured or released.
+
+###  4.3. <a name='AbilitytouselocalconfigsviaFinderonmacOS'></a>Ability to use local configs via Finder on macOS
+
+A new `--working-dir <path>` command line parameter has been introduced to make
+it possible to launch DOSBox Staging using local configs from the Finder on
+macOS. Previously, you had to use DOSBox from the command line to achieve this.
+ 
+**How to use it:**
```

Comment:
Please use the next heading, `####` in this case, and remove the `:` from the end.